brew_wizard_ui/src/app/recipes/recipe-list/recipe-list.component.scss

```diff
@@ -84,4 +94,58 @@
 
 .no-padding {
   padding: 0;
+}
+
+
+
+@media screen and (max-width: 420px) {
+
+  h3 {
+    font-size: 1.5rem;
+  }
+  
+  .table-header {
+    padding: 25px;
+    padding-bottom: 5px;
+  }
+
+  .card-header {
+    padding-top: 0.5em;
+    padding-bottom: 0.5em; 
+  }
+
+  .status {
+    padding-left: 30%;
+  }
+
+  .mini-row {
+    h5 {
+      font-size: smaller;
+      margin-bottom: -5px;
+    }
+    height: 20px;
```

Comment:
I like the look if we drop the height on mini-row here, and change the recipe name from smaller to medium (or pick a number). Or small size to 70% w/ recipe name as smaller,  I think the recipe name needs a higher significance than the creator name. the smaller/70% I think looks alright on mobile, on desktop it looks too small.